### PR TITLE
과제3 OTX를 이용한 model training - 김기범

### PR DESCRIPTION
**Pull request by bluelitma**
submit hw3

### CODE REVIEW

File: class02/homework/kimgibeom/hw3/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+./splitted_dataset/train/good: 24
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S| |
+|EfficientNet-B0|1.0|94.9570675358|0:00:10.531075|64|0.0049|----|----|
+|DeiT-Tiny| 
+|MobileNet-V3-large-1x|1.0|146.049855287|0:00:06.846977|64|0.0058|----|----|
```
Comment (JerryJLee) [inline class02/homework/kimgibeom/hw3/README.md:21]: 왜 결과가 2개 뿐이죠?

Comment (bluelitma) [reply class02/homework/kimgibeom/hw3/README.md:21]: 1개 이상만 하시라고 하셔서 시간 되는데로 2개만 했습니다.

**Review by JerryJLee — APPROVED** [review]
